src/main/java/org/jabref/gui/SendAsEMailAction.java

```diff
@@ -107,19 +111,30 @@ private String sendEmail() throws Exception {
             }
         }
 
-        String mailTo = "?Body=".concat(rawEntries.toString());
-        mailTo = mailTo.concat("&Subject=");
-        mailTo = mailTo.concat(preferencesService.getExternalApplicationsPreferences().getEmailSubject());
-        for (String path : attachments) {
-            mailTo = mailTo.concat("&Attachment=\"").concat(path);
-            mailTo = mailTo.concat("\"");
-        }
-
-        URI uriMailTo = new URI("mailto", mailTo, null);
+        URI uriMailTo = getUriMailTo(rawEntries, attachments);
 
         Desktop desktop = Desktop.getDesktop();
         desktop.mail(uriMailTo);
 
         return String.format("%s: %d", Localization.lang("Entries added to an email"), entries.size());
     }
+
+    private URI getUriMailTo(StringWriter rawEntries, List<String> attachments) throws URISyntaxException {
+        StringBuilder mailTo = new StringBuilder();
+        if (action == StandardActions.SEND_TO_KINDLE) {
+            mailTo.append(preferencesService.getExternalApplicationsPreferences().getKindleEmail());
+            mailTo.append("?Subject=").append(Localization.lang("Send to Kindle"));
```

Comment:
I would also add "Send to kindle" as email body. Otherwise, an empty email could be rejected. Who knows...

Reply:
Sure, will add that. I did try it out using my Kindle email and it does not require a subject or body, but agreed that it would be best to future proof ourselves by adding both anyways 😄

Reply:
Checking for attachments is enough. Enforced Email configuration could be too hard for newcomers to JabRef.